pages/index.js

```diff
@@ -16,8 +18,19 @@ export default class Home extends Component {
             <h1>{title}</h1>
             <HomeContent />
           </article>
+          <CommunitiesCarousel communities={this.props.communities} />
         </Layout>
       </>
     );
   }
 }
+
+export async function getStaticProps({ params }) {
```

Comment:
we can also leave it as is, but if you have time we can have a talk about react hooks & we would also make this Home component not class anymore + use hooks to get info inside. Other than this observation everything else looks great! good job!

Reply:
@nutzaalex I think that this is a correct use of getStaticProps. It'll result in proper static page generation, which an internal query wouldn't. This is a feature of next.js we've not had the opportunity to use in G4. It is very cool and well-worth reading about.